src/payment/AlertCodeMessages.jsx

```diff
@@ -74,3 +74,37 @@ export const TransactionDeclined = () => (
     />
   </>
 );
+
+export const CaptureKeyTimeoutExplanation = () => (
+  <>
+    <FormattedMessage
+      id="payment.messages.key.timeout.body"
+      defaultMessage="For security, your credit card information will then need to be re-entered to complete your purchase."
```

Comment:
I wonder if it's worth being very explicit with something like "if the purchase is not completed by the timer... then for security, your credit card information will..." or something with "this page will be refreshed and you'll need to re-enter..."

Reply:
Perhaps; I'm trying to keep the messages short, though.  Might be something to test later.